src/lib/validationRules/noExplicitDefaults/noExplicitDefaults.ts

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { RuleResult, ValidationRule, ValidationContext } from "../../types";
+
+const noExplicitDefaults: ValidationRule = {
+  rule: {
+    type: "local",
+    name: "No explicit defaults needed",
+    description:
+      "If a field is set to its default value, this is not necessary and can be left out, unless useful for clarity of understanding.",
+  },
+  check: async (context: ValidationContext) => {
+    // helper to create info messages
+    const createValidationResultForDefaultValue = (
+      fieldName: string
```

Comment:
Id you take a `value` or `document` option here, then you can lift this function up, and do:

```
results.push(createValidationResultForDefaultValue("grant_types", context.document));
```